### PR TITLE
Some improvements to german language files

### PR DESCRIPTION
**Pull request by ErrorCraftLP**
Also removed some unused entries to the removed oxygen pipes. Line numbers are identical now :)

### CODE REVIEW

File: src/main/resources/assets/galacticraftcore/lang/de_DE.lang [hunk 1]
```diff
@@ -593,7 +592,7 @@ tile.dummyblock.space_station_base.name=Raumstationsbasis
 tile.electric_furnace_tier1.description=Der elektrische Ofen ist eine schnellere Alternative zu dem Kohleofen.
 tile.electric_furnace_tier2.description=Der elektrische Lichtbogenofen ist eine bessere und schnellere Alternative zu dem normalen und dem elektrischen Ofen: Verdoppelung von Erzen!
 tile.enclosed.aluminum_wire.name=Versiegelbares Aluminiumkabel
-tile.enclosed.heavy_aluminum_wire.name=Versiegelbares Schweres Aluminiumkabel #NOT IN THE US LANG FILE o_O
+tile.enclosed.heavy_aluminum_wire.name=Versiegelbares Schweres Aluminiumkabel
```
Comment (ErrorCraftLP) [inline src/main/resources/assets/galacticraftcore/lang/de_DE.lang]: https://github.com/micdoodle8/Galacticraft/blob/master/src/main/resources/assets/galacticraftcore/lang/en_US.lang#L600

File: src/main/resources/assets/galacticraftplanets/lang/de_DE.lang [hunk 2]
```diff
@@ -291,8 +291,6 @@ tile.mars_machine.2.name=Start-Controller
 tile.mars_machine.4.name=Gasverflüssiger
 tile.mars_machine.5.name=Methansynthesegerät
 tile.mars_machine.6.name=Wasser-Elektolysegerät
-tile.mars_stairs_brick.name=Mars Dungeonziegel Stufen
```
Comment (ErrorCraftLP) [inline src/main/resources/assets/galacticraftplanets/lang/de_DE.lang]: Note: These entries already exist some lines above.

**Comment by radfast** [comment]
Hi @ErrorCraftLP  thank you for this, always appreciated

I can see you reviewed and updated 20 minutes ago, I'll leave this a couple more hours in case you want to make more changes before I merge it in?

**Comment by ErrorCraftLP** [comment]
You can merge it already if you want, I made these comments in case someone wondered why I removed those lines.

**Pull request merged**